boofilsic/settings.py

```diff
@@ -54,7 +62,12 @@
     'movies.apps.MoviesConfig',
     'music.apps.MusicConfig',
     'games.apps.GamesConfig',
+    'sync.apps.SyncConfig',
+    'collection.apps.CollectionConfig',
+    'timeline.apps.TimelineConfig',
     'easy_thumbnails',
+    'user_messages',
+    'django_slack',
```

Comment:
Maybe a mistake at my side, but I don't see the use of this anywhere else?

Reply:
user_messages is used to send a notification fwhen background tasks completes (e.g. export)

django_slack can be configured in `settings.py` to send notification to an internal instance when 500. I have this in my prod settings: 
```
SLACK_TOKEN = '...'
SLACK_CHANNEL = 'xxx'

LOGGING = {
    'handlers': {
        'slack': {
            'level': 'ERROR',
            'class': 'django_slack.log.SlackExceptionHandler',
        },
    },
```

Reply:
user_messages was accidentally selected, makes sense, thanks